smoke-tests/images/servlet/build.gradle.kts

```diff
@@ -36,6 +36,10 @@ val targets = mapOf(
     // running configure.sh is failing while building the image with Java 19
     ImageTarget(listOf("21.0.0.10"), listOf("hotspot"), listOf("8", "11", "17"), mapOf("release" to "2021-09-20_1900")),
     ImageTarget(listOf("21.0.0.10"), listOf("openj9"), listOf("8", "11", "17"), mapOf("release" to "2021-09-20_1900")),
```

Comment:
I think we can drop this in favor of `21.0.0.12`

```suggestion
```

Reply:
Should I remove them in a follow up PR after we switched to the new images? If I remove them now, will it break existing tests?

Reply:
it's confusing, but it won't break tests

the tests will continue using the old tag until you update the tag in #7857:

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/8a7e8a8ca203a3ffbfd4baeb322213f600bc05de/smoke-tests/src/test/groovy/io/opentelemetry/smoketest/AppServerTest.groovy#L57

Reply:
It is ok to remove them now. Tests use images with a specific tag, they won't automatically switch to the new images.